futurenet/horizon/etc/stellar-captive-core.cfg

```diff
@@ -6,7 +6,9 @@ PUBLIC_HTTP_PORT=false
 # To avoid conflicts with the core instance
 PEER_PORT=11725
 DATABASE="__DATABASE__"
-__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__
+
+ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__
+ENABLE_DIAGNOSTICS_FOR_TX_SUBMISSION=__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__
```

Comment:
Adding `ENABLE_DIAGNOSTICS_FOR_TX_SUBMISSION ` to the captive stellar-core is causing Horizon to fail to startup with the following error:
```
error generating captive core configuration: invalid captive core toml file: could not unmarshal captive core toml: these fields are not supported by captive core: ["ENABLE_DIAGNOSTICS_FOR_TX_SUBMISSION"]
```

As far as I can tell that config key was added to horizon's check two months ago, well before the most recent horizon v2.28.3 release by @2opremio: https://github.com/stellar/go/commit/3ca501f09055d64f6721cb2531df7673db09ffed.

I'm guessing that version of horizon is on a fork from master that didn't get that change.

Reply:
And, actually, I guess it doesn't make sense to enable the submission diagnostics on Horizon's captive core, because I don't think Horizon passes the diagnostics back to the caller. So I shouldn't be enabling it on the Horizon captive core anyway.